common/src/test/scala/uk/ac/wellcome/models/ConceptTest.scala

```diff
@@ -0,0 +1,69 @@
+package uk.ac.wellcome.models
+
+import org.scalatest.{FunSpec, Matchers}
+import org.scalatest.prop.TableDrivenPropertyChecks._
+import uk.ac.wellcome.test.utils.JsonTestUtil
+import uk.ac.wellcome.utils.JsonUtil._
+
+class ConceptTest extends FunSpec with Matchers with JsonTestUtil {
+
+  val testCases = Table(
```

Comment:
Same as in the other PR. I don't really like this usage of property testing to shrink test cases. If all of the elements in this table are relevant tests cases then that should be made explicit by separating them in different tests

Reply:
There are a few tests similar to this and I think I might be missing something: they seem to verify that the case classes serialize and deserialize. Is this testing/verifying anything else?

Reply:
No really. to be honest I wonder whether they even need to exist. I don't think all of the model classes have them (I don't think I added them for Agent). It's also tested again in `IdentifiedWorkTest` and `UnidentifiedWorkTest`